modules/internal/ChapelArray.chpl

```diff
@@ -2343,53 +2343,56 @@ module ChapelArray {
     return dom[(...ranges)];
   }
 
-  proc #(dom: domain, counts: integral) where isRectangularDom(dom) && dom.rank == 1 {
+  operator #(dom: domain, counts: integral) where isRectangularDom(dom) &&
```

Comment:
I suspect that most of the operators defined in this file should be methods. You should probably leave that for future work though.

Reply:
I believe I encountered syntax errors when trying it the first time, but I can definitely revisit it

Reply:
My reaction was the same as Michael's: That these would make more sense as methods (but it might not practically matter since it's an always-used module).